mempool/reactor.go

```diff
@@ -103,6 +103,7 @@ type PeerState interface {
 // Send new mempool txs to peer.
 func (memR *MempoolReactor) broadcastTxRoutine(peer p2p.Peer) {
 	if !memR.config.Broadcast {
+		memR.Logger.Info("Tx broadcasting is disabled")
```

Comment:
Shouldn't we output this just once during initialisation?

Reply:
Yes

Reply:
added config to github issue template. we should not look at the logs every time we suspect a person changed its config file.

Reply:
https://github.com/tendermint/tendermint/pull/1764/commits/3d30a4294389bdaa9bb0a49a772d315b7fe89aad

Reply:
if we still want to do this, why not just dump the whole config file upon start? this sounds like a weird thing to do though

Reply:
what about an endpoint that can be queried? Or is this rather to have easier access to relevant information when people paste logs? In this case, if you log the config at boot a user might not have the full log at hand (rotated, scrolled out of cmdline-buffer, ...)

Reply:
@ebuchman what's the motivation behind this?